core/application/impl/configuration_storage_impl.hpp

```diff
@@ -19,11 +19,16 @@ namespace kagome::application {
     ~ConfigurationStorageImpl() override = default;
 
     const primitives::Block &getGenesis() const override;
+    std::vector<libp2p::peer::PeerInfo> getPeersInfo() const override;
+    std::vector<crypto::SR25519PublicKey> getSessionKeys() const override;
+    std::vector<crypto::ED25519PublicKey> getAuthorities() const override;
+
+    uint32_t getExtrinsicApiPort() const override;
```

Comment:
Maybe it is better to use unsigned short type (probably it was me who suggested to use uint32, sorry for that).
In [boost](https://www.boost.org/doc/libs/1_71_0/doc/html/boost_asio/reference/ip__basic_endpoint/port.html) this type is used